### ISSUE TITLE
Dispatcher settings are not copied on blueprints

### ISSUE THREAD

**Issue by Ocean-Phantom** [dvilker/FactorioRailLogisticsDispatcher#15]
I was testing out some prints i made, and it seems like the dispatcher entity is always set to "Off" mode after pasting blueprints.
For instance, the blueprint below is supposed to have 3 stations for refueling. And two more stations will request Uranium item and Rocket Fuel item. But when I tried pasting it down, all the settings placed are cleared and reset.

[new.txt](https://github.com/dvilker/FactorioRailLogisticsDispatcher/files/10328481/new.txt)

**Comment by dvilker** [comment]
Yes, it is clear that there is no information about the dispatcher settings in the blueprint. Unfortunately, the Factorio API does not provide an easy direct way to save settings to a blueprint. These manipulations are done indirectly, and other mods can completely interfere.

I have an idea how to fix this, I'll try later.

**Comment by Ocean-Phantom** [comment]
Really? Is it possible to just save just the supply or request information via the hidden constant combinator? Things like allowed trains and network bit are fine not to be copied. But I think having the delivery details be preserved should be the priority for blueprints.
